server/proto/types.go

```diff
@@ -1,3 +1,12 @@
 package ChannerProto
 
-type UUID uint64
+import (
+	"github.com/umegaya/yue"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/yue (can't find import: github.com/umegaya/yue)